x-pack/plugins/cloud_security_posture/public/pages/findings/findings_distribution_bar.tsx

```diff
@@ -0,0 +1,159 @@
+
+export const FindingsDistributionBar = (props: Props) => (
+  <div>
+    <Counters {...props} />
+    <EuiSpacer size="s" />
+    <DistributionBar {...props} />
+  </div>
+);
+
+const Counters = ({ pageStart, pageEnd, total, failed, passed }: Props) => (
+  <EuiFlexGroup justifyContent="spaceBetween">
+    <EuiFlexItem>
+      <CurrentPageOfTotal pageStart={pageStart} pageEnd={pageEnd} total={total} />
+    </EuiFlexItem>
+    <EuiFlexItem
+      css={css`
+        align-items: flex-end;
+      `}
+    >
+      <PassedFailedCounters passed={passed} failed={failed} />
+    </EuiFlexItem>
+  </EuiFlexGroup>
+);
+
+const PassedFailedCounters = ({ passed, failed }: Pick<Props, 'passed' | 'failed'>) => {
+  const { euiTheme } = useEuiTheme();
+  return (
+    <div
+      css={css`
+        display: flex;
+      `}
+    >
+      <Counter
+        label={i18n.translate('xpack.csp.findings.distributionBar.totalPassedLabel', {
+          defaultMessage: 'Passed',
+        })}
+        color={euiTheme.colors.success}
+        value={passed}
+      />
+      <Counter
+        label={i18n.translate('xpack.csp.findings.distributionBar.totalFailedLabel', {
+          defaultMessage: 'Failed',
+        })}
+        color={euiTheme.colors.danger}
+        value={failed}
+      />
+    </div>
+  );
+};
+
+const CurrentPageOfTotal = ({
+  pageEnd,
+  pageStart,
+  total,
+}: Pick<Props, 'pageEnd' | 'pageStart' | 'total'>) => (
+  <EuiTextColor color="subdued">
+    {total > 0 && (
+      <FormattedMessage
+        id="xpack.csp.findings.distributionBar.showingPageOfTotalLabel"
+        defaultMessage="Showing {pageStart}-{pageEnd} of {total} Findings"
+        values={{
+          pageStart: <b>{pageStart}</b>,
+          pageEnd: <b>{pageEnd}</b>,
+          total: <b>{formatNumber(total)}</b>,
+        }}
+      />
+    )}
+  </EuiTextColor>
+);
+
+const DistributionBar: React.FC<Props> = ({ passed, failed }) => {
+  const { euiTheme } = useEuiTheme();
+  return (
+    <div
+      css={css`
+        display: flex;
+        align-items: center;
+        width: 100%;
+        height: 8px;
+        background: ${euiTheme.colors.subdued};
+      `}
+    >
+      <div
+        css={css`
+          flex: ${passed};
+          background: ${euiTheme.colors.success};
+          height: 100%;
+        `}
+      />
+      <div
+        css={css`
+          flex: ${failed};
+          background: ${euiTheme.colors.danger};
+          height: 100%;
+        `}
+      />
+    </div>
+  );
+};
+
+const Counter = ({ label, value, color }: { label: string; value: number; color: string }) => {
+  const { euiTheme } = useEuiTheme();
+  return (
+    <div
+      css={css`
+        display: flex;
+        align-items: center;
+        &:not(:last-child) {
+          margin-right: ${euiTheme.size.l};
+        }
+        span:not(:last-of-type) {
+          margin-right: ${euiTheme.size.s};
+        }
```

Comment:
might be better if the consumer of `Counter` will handle placement and margins for it